0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/json4s/StringSerializers.scala

```diff
@@ -23,15 +23,19 @@ object StringSerializers {
 
   object FormatSerializer extends CustomSerializer[Format](_ => (
     {
-      case JString(format) if format == "ipv4" => Ipv4Format
-      case JString(format) if format == "ipv6" => Ipv6Format
-      case JString(format) if format == "uri" => UriFormat
-      case JString(format) if format == "email" => EmailFormat
-      case JString(format) if format == "hostname" => HostNameFormat
-      case JString(format) if format == "date-time" => DateTimeFormat
-      case JString(format) if format == "date" => DateFormat
-      case JString(format) => CustomFormat(format)
-      case x => throw new MappingException("Format must be string")
+      case JObject(list) =>
```

Comment:
Doesn't match `JString|JInt|JArray|etc...`. It means it'll throw unexpected exception and serializers won't work as expected.